### PR TITLE
feat: 🎸 add calendar

### PR DESCRIPTION
**Pull request by TiffL**
### Pull Request
---

#### PR Summary
Add new calendar to replace google calendar on homepage.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [x] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [ ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Review by fernandogelin — APPROVED** [review]

**Comment by hkershaw-brown** [comment]
Hello, it looks like events are only being grabbed for the current month.  e.g. for this week on 

https://brown-ccv.github.io/ccv-calendar/

there is no office hours event on Friday.  I am guessing this is because Friday is May 1st.

**Comment by hkershaw-brown** [comment]
The more info linking to events.brown.edu is really nice.

**Comment by mcmcgrath13** [comment]
This is awesome @TiffL !!!

A few nit picks/bugs:

1. Right now, friday is shown as 4/1 in week view, but it should be 5/1, also per @hkershaw-brown we need to grab the events there.
2. Can we move the see full calendar link closer to the calendar? I didn't notice it and then was looking for a next week button and couldn't find that either.

File: themes/gb-theme/layouts/partials/blocks/events.html [hunk 1]
```diff
@@ -22,8 +22,9 @@ <h1 class="py-8 h1-home">Upcoming Events</h1>
     </div>
   </div>
 </section>
-<div class="container-fluid pb-4">
+<div class="container-fluid">
   <section class="bg-white row d-flex flex-row justify-content-center">
-    <iframe src="https://calendar.google.com/calendar/b/1/embed?height=400&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FNew_York&amp;src=Y2N2LWJvdEBicm93bi5lZHU&amp;color=%2322AA99&amp;showTitle=0&amp;showNav=0&amp;showDate=1&amp;showTabs=0&amp;showCalendars=0&amp;showTz=0&amp;showPrint=0&amp;title=CCV%20Events&amp;mode=WEEK" style="border-width:0" width="1000" height="400" frameborder="0" scrolling="no"></iframe>
+    <iframe src="https://brown-ccv.github.io/ccv-calendar/" style="border-width:0" width="1500" height="910" frameborder="0" scrolling="no"></iframe>
+    <!--<iframe src="https://calendar.google.com/calendar/b/1/embed?height=400&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FNew_York&amp;src=Y2N2LWJvdEBicm93bi5lZHU&amp;color=%2322AA99&amp;showTitle=0&amp;showNav=0&amp;showDate=1&amp;showTabs=0&amp;showCalendars=0&amp;showTz=0&amp;showPrint=0&amp;title=CCV%20Events&amp;mode=WEEK" style="border-width:0" width="1000" height="400" frameborder="0" scrolling="no"></iframe>-->
```
Comment (mcmcgrath13) [inline themes/gb-theme/layouts/partials/blocks/events.html]: delete this comment

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

**Comment by TiffL** [comment]
Hi all, thanks! I've made the changes

**Review by mcmcgrath13 — APPROVED** [review]
This is awesome! 🚀

**Review by paulstey — APPROVED** [review]
This looks absolutely phenomenal!! Thank you!!

**Comment by paulstey** [comment]
@fernandogelin & @mcmcgrath13: Can we go ahead and merge this guy?

**Comment by mcmcgrath13** [comment]
@paulstey it's still in the waiting period, otherwise yes!

**Pull request merged**